homeassistant/components/gogogate2/cover.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         door: AbstractDoor,
     ) -> None:
         """Initialize the object."""
+        super().__init__(data_update_coordinator)
         self._config_entry = config_entry
         self._data_update_coordinator = data_update_coordinator
```

Comment:
```suggestion
```

You can drop this as well and use `self.coordinator` since `CoordinatorEntity` will set it